superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -116,16 +118,16 @@ const getEntityTitle = (entity: ActivityObject) => {
   return entity.item_title || UNTITLED;
 };
 
-const getEntityIconName = (entity: ActivityObject) => {
-  if ('sql' in entity) return 'sql';
+const getEntityIcons = (entity: ActivityObject) => {
```

Comment:
I think this being plural is confusing. Seems to indicate we are returning multiple icons.

Reply:
Agree.